server/utils/formatter.js

```diff
@@ -82,7 +82,10 @@ export function formatPayload(automation) {
  * @returns {Array} Formated automation to be returned by the API
  */
 export function formatAutomationResponse(payload) {
-  return payload.map(automation => formatPayload(automation));
+  if (!payload) {
```

Comment:
```suggestion
  if (payload) {
```
This should be flipped like so instead, else the `payload` is never formatted, making the details modal show no details